packages/itwinui-react/src/core/ComboBox/ComboBox.test.tsx

```diff
@@ -1063,3 +1063,12 @@ it('should be customizable', async () => {
   const statusContent = getByText('My message');
   expect(statusContent).toHaveClass('custom-message-class');
 });
+
+it('should allow passing ref to ComboBox', () => {
+  const selectRef = React.createRef<HTMLElement>();
+  render(
+    <ComboBox options={[{ label: 'Item 1', value: 1 }]} ref={selectRef} />,
+  );
+  console.log(selectRef.current);
+  expect(selectRef?.current).toHaveAttribute('role');
```

Comment:
use a custom attribute because
1. checking for `role` is not sufficient
2. implementation details should not be tested